lib/ansible/modules/files/unarchive.py

```diff
@@ -548,7 +548,7 @@ def is_unarchived(self):
             group = gid = None
             try:
                 group = grp.getgrgid(st.st_gid).gr_name
-            except:
+            except (KeyError, ValueError, OverflowError):
```

Comment:
What scenario will raise `OverflowError`?

Reply:
For example
```
>>> grp.getgrgid(10**10)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
OverflowError: gid is less than minimum
```